modules/qc.nf

```diff
@@ -141,3 +141,32 @@ process multiqc {
     ${params.multiqcOptions}
     """
 }
+
+process fastqcNanopore {
+    publishDir "${params.outdir}/QCStats/${task.process.replaceAll(":","_")}", mode: 'copy', overwrite: true
+
```

Comment:
@henningonsbring input files missing?

Reply:
As it is written the code is working, fastqc is runned and multiqc then started when fastqc is done. For this to work I need to set `fastqPath` to `fastqPath = "${params.outdir}/../fastq/*/*fastq.gz"` in the config. I have very limited experience with nextflow, is there another way you want this to be done that is more elegant?

Reply:
Hi again,
I think if you add input files like this in the `fastqcNanopore` process, it will work. fastq path is already added in `Nanopore config` file. I tested locally with new the container for Nanopore -
1. `qc.nf`
```
process fastqcNanopore {
    
    publishDir "${params.outdir}/QCStats/${task.process.replaceAll(":","_")}", mode: 'copy', overwrite: true

    input:
    path fastq
    
    output:
    path("*.zip") , emit: zip
    path("*.html") , emit: html

    """
    fastqc ${fastq}/*.fastq --format fastq --threads ${task.cpus} --dir ${params.tmpdir} --outdir .
    """
}
```
2. `articNcovNanopore.nf`. 

```
      fastqcNanopore(ch_runFastqDirs)
      
      multiqcNanopore(fastqcNanopore.out.zip)
```

Also, needs to add this on process `versions` (`analysis.nf` module) for fastqc and multiqc versions - 

```
...
    else if ( params.medaka || params.nanopolish )
        """
        artic --version > version_artic.txt
        fastqc -v > version_fastqc.txt
        multiqc --version > version_multiqc.txt
        bwa > version_bwa.txt 2>&1 || true
        mafft --version > version_mafft.txt 2>&1
        medaka --version > version_medaka.txt
...

```